dashboard/src2/components/AlertAddPaymentMode.vue

```diff
@@ -1,7 +1,10 @@
 <template>
 	<AlertBanner
-		title="Add a payment mode to start creating sites, bench groups, and more."
-		type="warning"
+		:title="
+			title ??
+			'Add a payment mode to start creating sites, bench groups, and more.'
+		"
+		:type="type ?? 'warning'"
```

Comment:
what's the scenario where the `type` changes?

Reply:
Warning shows up as a yellow banner. I want a blue banner inside the site's dashboard. So passing "info" as type in there to achieve the same.